reference/release-notes/1.27.0.md

```diff
@@ -0,0 +1,63 @@
+
+### Other
+
+* The Android WebView has been updated to [138.0.7204.63](https://chromereleases.googleblog.com/2025/06/chrome-for-android-update_30.html).
+
+## Deprecations and removed functionality
+
+* Support for Juju 2.9 was deprecated in 1.25.1 and is removed with this release. Instead, upgrade to the latest Juju version, 3.6. See {ref}`sec-juju-version-requirements` for more information.
+* Support for kernels older than 6.8 was deprecated in 1.25.0 and is removed with this release.
+
+## Known issues
+
+See our [open bugs in Launchpad](https://bugs.launchpad.net/anbox-cloud/?field.searchtext=&orderby=-importance&field.status%3Alist=NEW&field.status%3Alist=CONFIRMED&field.status%3Alist=TRIAGED&field.status%3Alist=INPROGRESS&field.status%3Alist=DEFERRED&field.status%3Alist=FIXCOMMITTED&field.status%3Alist=INCOMPLETE_WITH_RESPONSE&field.status%3Alist=INCOMPLETE_WITHOUT_RESPONSE&assignee_option=any&field.assignee=&field.bug_reporter=&field.bug_commenter=&field.subscriber=&field.structural_subscriber=&field.milestone%3Alist=102837&field.tag=&field.tags_combinator=ANY&field.has_cve.used=&field.omit_dupes.used=&field.omit_dupes=on&field.affects_me.used=&field.has_patch.used=&field.has_branches.used=&field.has_branches=on&field.has_no_branches.used=&field.has_no_branches=on&field.has_blueprints.used=&field.has_blueprints=on&field.has_no_blueprints.used=&field.has_no_blueprints=on&search=Search) that are planned to be fixed for the 1.27.1 release, next month.
+
+The following CVEs are assessed for severity and impact but not included with the 1.27.0 release. They will be included in the subsequent releases:
+
+* [CVE-2025-22441](https://source.android.com/docs/security/bulletin/2025-08-01) for Android 13, 14, 15
+* [CVE-2025-48533](https://source.android.com/docs/security/bulletin/2025-08-01) for Android 13, 14, 15
+
+## Bug fixes
+
+* [LP 2110219](https://bugs.launchpad.net/anbox-cloud/+bug/2110219) When running an Anbox instance with an NVIDIA GPU and utilizing it for rendering and video encoding, the `lxcfs` process on the host shows an unusually high CPU consumption (~60% for a single Anbox instance).
+* [LP 2113943](https://bugs.launchpad.net/anbox-cloud/+bug/2113943) Easy-RSA does not handle URLs with protocols correctly.
+* [LP 2116159](https://bugs.launchpad.net/anbox-cloud/+bug/2116159) At times, setting the AMS config `cpu.limit_mode` to `scheduler` prevents instances from starting.
+* [LP 2117094](https://bugs.launchpad.net/anbox-cloud/+bug/2117094) For Android 15, trying to access developer options in the *Settings* app causes an OS-level crash.
+* [LP 2097514](https://bugs.launchpad.net/anbox-cloud/+bug/2097514) When using the Android 14 images, trying to reset mobile network settings causes an OS-level crash.
+* [LP 2112562](https://bugs.launchpad.net/anbox-cloud/+bug/2112562) The Android interface is unresponsive because of an Android crash. This occurs only occasionally but requires a full restart of Anbox Cloud to recover.
```

Comment:
We need to reword this bug. e.g:
A crash in HWComposer caused the Android system to restart.